src/uproot/containers.py

```diff
@@ -876,20 +964,34 @@ def awkward_form(self, file, context):
         )
 
     def read(self, chunk, cursor, context, file, selffile, parent, header=True):
+        forth = False
+        forth_obj = None
+        fcode_pre = []
+        fcode_post = []
+        temp = None
+        if "forth" in context.keys():
+            forth = True
+            forth_obj = context["forth"]
         if self._header and header:
             start_cursor = cursor.copy()
             (
                 num_bytes,
                 instance_version,
                 is_memberwise,
             ) = uproot.deserialization.numbytes_version(chunk, cursor, context)
+            if forth:
+                temp_jump = cursor._index - start_cursor._index
+                if temp_jump != 0:
+                    fcode_pre.append(f"{temp_jump} stream skip\n")
         else:
             is_memberwise = False
 
         # Note: self._values can also be a NumPy dtype, and not necessarily a class
         # (e.g. type(self._values) == type)
         _value_typename = _content_typename(self._values)
         if is_memberwise:
+            if forth:
+                raise NotImplementedError
```

Comment:
These should eventually give a reason: `NotImplementedError` because of memberwise serialization.